components/EditTwitterAccount.js

```diff
@@ -189,17 +189,17 @@ class EditTwitterAccount extends React.Component {
               <FormattedMessage id="connectedAccounts.twitter.settings" defaultMessage="Settings" />
             </summary>
             {this.notificationTypes.map(this.renderNotification)}
+            <Row>
+              <Col sm={3} />
+              <Col sm={9}>
+                {this.state.isModified && (
+                  <SmallButton className="default" bsStyle="primary" onClick={this.onClick}>
+                    <FormattedMessage id="save" defaultMessage="save" />
+                  </SmallButton>
+                )}
+              </Col>
+            </Row>
```

Comment:
What's the purpose if this change?

Reply:
I see.

> Fix the settings not opening issue

What was that?

Reply:
This is how `details` in html work - `summary` and `content` should be inside the `details` tag
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details